testcases/distro-smoke.yaml

```diff
@@ -0,0 +1,82 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set test_timeout = 120 %}
+{% block metadata %}
+  {{ super() }}
+{% endblock metadata %}
+
+{% set test_name = test_name | default("distro-smoke") %}
+
+{% block test_target %}
+  {{ super() }}
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/smoke/smoke.yaml
+      params:
+        TESTS: {{SMOKE_TESTS}}
+      name: linux-smoke
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/usb-smoke/usb-smoke-test.yaml
+      name: linux-usb-smoke
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/toolchain-smoke/toolchain-smoke.yaml
+      name: linux-toolchain-smoke
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/device-tree/device-tree.yaml
+      name: linux-device-tree
+    - repository: {{ TEST_DEFINITIONS_REPOSITORY }}
+      from: git
+      path: automated/linux/meminfo/meminfo.yaml
+      name: meminfo
+
```

Comment:
what's the reason for the newlines in random places in this file?

Reply:
Any reason, at all, if you like i Can remove.

Reply:
yeah please drop it.

Reply:
@roxell Done.